VacationRental.Api/Controllers/RentalsController.cs

```diff
@@ -27,17 +37,79 @@ public RentalViewModel Get(int rentalId)
         }
 
         [HttpPost]
+        //[Route("vacationrental/rentals")]
```

Comment:
I added this comments because I am a little bit confused about changing the endpoint or not ? 
![image](https://user-images.githubusercontent.com/101488182/169709237-c9caa56b-0f08-46f4-a14f-e7680ded44e2.png)